source/common/router/config_impl.cc

```diff
@@ -420,11 +420,11 @@ RouteEntryImplBase::RouteEntryImplBase(const VirtualHostImpl& vhost,
         std::make_unique<TlsContextMatchCriteriaImpl>(route.match().tls_context());
   }
 
-  // Only set include_vh_rate_limits_ to true if the rate limit policy for the route is empty
-  // or the route set `include_vh_rate_limits` to true.
+  // Returns true if include_vh_rate_limits is explicitly set to true otherwise it defaults to false
+  // which is similar to VhRateLimitOptions::Override and will only use virtual host rate limits if
+  // the route is empty
```

Comment:
Is this going to be a behavior change for existing users who do not use per-route configs? I'm worried we might be changing something here but I'm a little fuzzy on the details?

Reply:
OK cool thanks for the explanation!